style.css

```diff
@@ -0,0 +1,286 @@
+* {
+  font-family: "Poppins", sans-serif;
+}
+
+body {
+  position: absolute;
+  width: 375px;
```

Comment:
- [ ] You have an X overflow, this width could be the issue or maybe another width of another class or element, please check your code. [Here is the live demo](https://rawcdn.githack.com/tacettinkustu/My-Portfolio-Website/b9b9fbaaba46b7f48d90d09aa13c165708d37787/index.html) of your page. [This link](https://stackoverflow.com/questions/1165497/how-to-prevent-text-from-overflowing-in-css) could help you as well